bin/setup

```diff
@@ -1,28 +1,18 @@
 #!/usr/bin/env ruby
 require 'pathname'
 
-GEM_ROOT = Pathname.new File.expand_path('../../',  __FILE__)
+GEM_ROOT = Pathname.new(__dir__).join("..")
 
 Dir.chdir(GEM_ROOT) do
   if File.exist?("spec/manageiq")
     puts "== Updating manageiq sample app =="
     system "cd spec/manageiq && git pull"
   else
     puts "== Cloning manageiq sample app =="
-    system "git clone git@github.com:ManageIQ/manageiq.git --depth 1 spec/manageiq"
+    source = ENV["CI"] ? "https://github.com/ManageIQ/manageiq.git" : "git@github.com:ManageIQ/manageiq.git"
```

Comment:
I want to share the same script between users and Travis.  Travis doesn't have any ssh keys, so it can't use the SSH option.